lib/src/main/java/uk/gov/di/ipv/core/library/domain/IdentityClaim.java

```diff
@@ -1,9 +1,11 @@
 package uk.gov.di.ipv.core.library.domain;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
 
 import java.util.List;
 
+@EqualsAndHashCode
```

Comment:
Why do we need this?

Reply:
It's for the test I extended in [UserIdentityHandlerTest.java](https://github.com/alphagov/di-ipv-core-back/pull/308/files#diff-f99dd793b7e45d90a250ac427fb8fa617e8f8e2ffa6fca67435ce9b2d8f649d9R148). Without adding this it was comparing the objects identities, rather than their equalities.